java-library/pom.xml

```diff
@@ -4,7 +4,7 @@
 
   <groupId>com.microsoft.azure.functions</groupId>
   <artifactId>azure-functions-java-library-sql</artifactId>
-  <version>0.1.0</version> <!-- Update the dependency version in samples-java/pom.xml too if this version is updated. -->
+  <version>0.1.1</version> <!-- Update the dependency version in samples-java/pom.xml too if this version is updated. -->
```

Comment:
See inline comment 😄

Reply:
I was planning on sending another PR out to update the samples pom.xml + add new sample after the new library is released. But I guess releasing the library is a quick process so do you think it'd be better to just update the version in this PR too?

Reply:
Yeah, that's a bit tricky. But since it's quick I think we should get in the habit of updating them in step so we don't forget to do it. If that ends up being too annoying then we can remove this comment and just update our release docs to add a reminder to update the samples as well.